.github/workflows/ci.yml

```diff
@@ -8,7 +8,12 @@ jobs:
     strategy:
       matrix:
         php-version: [ '7.4', '8.0', '8.1' ]
-        symfony-version: ['^4.4', '^5.0']
+        symfony-version: ['^4.4', '5.4']
+        include:
+          - php-version: '8.0'
+            symfony-version: '6.0'
+          - php-version: '8.1'
+            symfony-version: '6.0'
```

Comment:
:information_source: SF6 requires PHP 8+ and this syntax excludes his version from being tested with PHP7